styles/index.scss

```diff
@@ -15,13 +15,28 @@ $heading-font: 'Nunito', 'Avenir', sans-serif;
   width: 100%;
 }
 
+.top-photo {
+  padding: 0px 0px 0px 15px;
+  position: absolute;
+  left: 47%;
+  overflow: hidden;
+  width: 45%;
+}
+
 .section{
-  width: 100%;
+  position: relative;
+  min-width: 300px;
+}
+
+#home-details{
+  width: 45%;
+  position: relative;
 }
 
 .home-photos img {
   width: 400px;
   max-width: 800px;
+  max-height: 500px;
```

Comment:
you're on the right track here with making sure that the images appear the same height, but because they scale themselves to fit the available horizontal space, their heights can still not appear equal
![Screenshot_20200629_134447](https://user-images.githubusercontent.com/17362949/86054519-279b8200-ba0f-11ea-8b5f-38c0a3c3da7d.png)
 when they are side-by side